aict-MERN/backend-of-aict-site-w-react/db.js

```diff
@@ -0,0 +1,25 @@
+// require('dotenv').config()
+import dotenv from 'dotenv'
+dotenv.config()
```

Comment:
aaah, here's dotenv! this should be the first line to be executed in your program, hence it should go at the top of your `app.js`!

Reply:
So it doesn't need to be in this file if it is in app.js?